keyboards/montsinger/rebound/keymaps/default/keymap.c

```diff
@@ -0,0 +1,159 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
```

Comment:
You can simplify layer management and save a good deal of firmware space by replacing this with an `enum`:

```
enum layer_names
{
  _QWERTY,
  _COLEMAK,
  _DVORAK,
  _LOWER,
  _RAISE,
  _ADJUST
};
```

By having the layer `defines` jump from 4 to 16 you're essentially creating "phantom layers" 5-15 which will add a lot of overhead.